curriculum/challenges/english/03-front-end-libraries/react/render-conditionally-from-props.english.md

```diff
@@ -14,8 +14,10 @@ In this challenge, you'll set up a child component to make rendering decisions b
 ## Instructions
 <section id='instructions'>
 The code editor has two components that are partially defined for you: a parent called <code>GameOfChance</code>, and a child called <code>Results</code>. They are used to create a simple game where the user presses a button to see if they win or lose.
+
```

Comment:
Delete this extra line as it is not needed.  A new paragraph is created by starting a new line.  When tested locally, you can see the extra space between.
```suggestion

```